pages/index.js

```diff
@@ -10,7 +10,7 @@ export default function Home() {
 
       <main>
         <h1 className="title">
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
+          Welcome to <a href="#" onClick={()=>window.open("/api/login","Login","width=500,height=800")}>Next.js!</a>
```

Comment:
You can merge from dev/login-page-ui i've added a redirect
Also add a function to check if logged in instead of ```isLoggedIn```